src/components/Setup/ProtocolCard.js

```diff
@@ -32,11 +133,22 @@ ProtocolCard.defaultProps = {
 
 ProtocolCard.propTypes = {
   selectProtocol: PropTypes.func,
-  onDelete: PropTypes.func.isRequired,
+  openDialog: PropTypes.func.isRequired,
+  deleteProtocol: PropTypes.func.isRequired,
   protocol: PropTypes.shape({
+    uuid: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
     description: PropTypes.string,
   }).isRequired,
 };
 
-export default ProtocolCard;
+function mapDispatchToProps(dispatch) {
+  return {
+    openDialog: bindActionCreators(dialogActions.openDialog, dispatch),
+    deleteProtocol: bindActionCreators(installedProtocolsActions.deleteProtocol, dispatch),
+  };
+}
+
+export { ProtocolCard as UnconnectedProtocolCard };
```

Comment:
By convention these haven't previously specifed `Unconnected`.

Reply:
> By convention these haven't previously specifed `Unconnected`.

I'm not following on this one. Can you explain? We've used `UnconnectedComponentName` throughout for exports where we just want to test the unconnected component, haven't we?

Reply:
Maybe there are some exceptions, but I think the convention has been:
```
export { ConcentricCircles };

export default ConcentricCircles;
```

Usage:
```
// unconnected
import { ConcentricCircles } from './ConcentricCircles';

// connected
import ConcentricCircles from './ConcentricCircles';
```

Reply:
Hmm. I actually think that's less clear. Its probably something we did in NC, but not elsewhere. Its pretty widespread though: https://github.com/codaco/Network-Canvas/search?q=Unconnected&unscoped_q=Unconnected

Unless you feel really strongly (in which case I would update all uses for consistency), I think this should stay the same.

Reply:
I mean, I think it'd make more sense to have `ConcentricCircles` and `ConnectedConcentricCircles`, but that's worse in my opinion! Since this is only required for testing, I don't think it matters either way. The convention in Architect is without `Unconnected`, and at least that's the case for a few components in NC.